src/main/java/io/xpring/xrpl/model/XRPMemo.java

```diff
@@ -19,33 +17,26 @@
   }
 
   /**
-   * An arbitrary hex value, conventionally containing the content of the memo.
+   * (Optional) An arbitrary hex value, conventionally containing the content of the memo.
    *
    * @return A byte array of arbitrary hex value, conventionally containing the content of the memo.
    */
-  @Nullable
   byte[] data();
 
   /**
-   * Hex value representing characters allowed in URLs.
-   * <p>
+   * (Optional) Hex value representing characters allowed in URLs.
    * Conventionally containing information on how the memo is encoded, for example as a MIME type.
-   * </p>
    *
    * @return A byte array of arbitrary hex value containing the characters allowed in URLs.
    */
-  @Nullable
   byte[] format();
```

Comment:
I'm honestly not sure what will happen here if you build an `XRPMemo` without specifying a `format`.  It might blow up and say that you have to set it.  I would test this out to confirm, and if true, do something like so:
```java
@Value.Default
default byte[] format() {
    return new byte[0];
}
```

Reply:
It will blow up.  However, the code that constructs an `XRPMemo` is in its `.from` method, which always remembers to set these fields... and even if they're not set on the protobuf object, we get a value such as `ByteString.EMPTY` which ends up constructing an empty `byte[]`.  However, I added the defaults for readability since there are a few layers to that logic that are not obvious at a glance.